components/dashicon/index.js

```diff
@@ -5,6 +5,11 @@ DO NOT EDIT THAT FILE! EDIT index-header.jsx and index-footer.jsx instead
 OR if you're looking to change now SVGs get output, you'll need to edit strings in the Gruntfile :)
 !!! */
 
+/**
+ * Internal dependencies
+ */
+import './style.scss';
```

Comment:
This file is generated, so this import would be overwritten next time the Dashicons is updated. It seems like the RTL flipping could be valuable to have in the base Dashicons repository, but in keeping it generic we'd not want to assume the presence of an `.rtl` body class. Would it be reasonable to detect RTL from JavaScript inside the component class and apply an inline style?

https://github.com/WordPress/dashicons/blob/master/sources/react/index-header.jsx
https://github.com/WordPress/dashicons/blob/master/sources/react/index-footer.jsx